### PR TITLE
Edit for readability and punctuation, fix spelling error

### PR DESCRIPTION
**Pull request by joe-crick**
Edit for readability and punctuation, fix spelling error

### CODE REVIEW

**Comment by daffl** [comment]
I merged #1964 first and now this one seems to have a conflict?

**Comment by joe-crick** [comment]
OK. I'll resolve, and resubmit.

**Comment by daffl** [comment]
No need to resubmit, just push to this branch once resolved (ideally rebased if possible). Thanks!

File: component/component.md [hunk 1]
```diff
@@ -359,6 +359,7 @@ The following tree combo lets people walk through a hierarchy, and select locati
 The secret to this widget is the viewModel's `breadcrumb` property, which is an array
 of items the user has navigated through, and `selectableItems`, which represents the children of the
 last item in the breadcrumb.  These are defined on the viewModel:
+>>>>>>> origin
```
Comment (daffl) [inline component/component.md]: I don't think this belongs here.

Comment (justinbmeyer) [inline component/component.md]: it doesn't.

**Comment by daffl** [comment]
Can you resolve the conflicts that came up after merging in minor?

**Comment by pYr0x** [comment]
can we merge this PR in version 2.3.4?

**Comment by justinbmeyer** [comment]
@pYr0x Yep.  Someone just has to do the merge.  cc @joe-crick ?

**Pull request closed without merging**